bvm/ballerina-core/src/main/java/org/ballerinalang/BLangProgramRunner.java

```diff
@@ -103,4 +135,17 @@ public static FunctionInfo getMainFunctionInfo(PackageInfo entryPkgInfo) {
         }
         return functionInfo;
     }
+
+    private static void executeListenPhase(ProgramFile programFile) {
+        PrintStream outStream = System.out;
+
+        ServerConnectorRegistry serverConnectorRegistry = new ServerConnectorRegistry();
+        programFile.setServerConnectorRegistry(serverConnectorRegistry);
+        serverConnectorRegistry.initServerConnectors();
+
+        outStream.println("Initiating service(s) in '" + programFile.getProgramFilePath() + "'");
```

Comment:
IMO, we shouldn't print anything to the outstream implicitly. That is Endpoint authors responsibility to put such logs. So shall we remove this?

Reply:
This has been there since the early days though. I just moved it to BLangProgramRunner.

Reply:
let's remove it.

Reply:
Ok, will do